.github/CONTRIBUTING.md

```diff
@@ -1,36 +1,92 @@
 # Contributing
 
+## Workflow
+
+1. Fork and clone this repository.
+2. Create a new branch in your fork based off the **main** branch.
+3. Make your changes.
+4. Commit your changes, and push them.
+5. Submit a Pull Request [here]!
+
+## Contributing to the guides
+
+The guides for all Sapphire Project use [mdx] formatting. At its core this is the exact same as regular markdown, and it supports all your regular GitHub flavoured markdown syntax. Where it differs from this regular markdown however, is that it _also_ supports inline JSX tags. These are used for advanced and optimal layout on `<docs site address pending>`. If you want to learn how and what kind of JSX you can embed into your [mdx] then please consult the [mdx documentation][]
+
+When writing guides there are some rules to follow:
+
+1. All files should have the `.mdx` file extension.
+2. Folder names are allowed to have spaces.
```

Comment:
Would it not be prevalent to, in line with filesystem conventions and ease of use, use lowercased folders with hyphens instead of spaces and then have the site convert that when reading the guides pages? I really do not think it's worth violating filesystem conventions for this, especially not when large companies do it so much already and get a fair bit of hate from dev communities for it.